### PR TITLE
Fix theme export

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | debug (develop)
| Description?  | Fix theme export
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1727
| How to test?  | 

See : https://github.com/PrestaShop/PrestaShop/pull/6868

### CODE REVIEW

File: controllers/admin/AdminThemesController.php [hunk 1]
```diff
@@ -224,11 +224,15 @@ public function renderView()
      */
     public function postProcess()
     {
+        global $kernel;
+
         if ('exporttheme' === Tools::getValue('action')) {
             $exporter = new ThemeExporter(
```
Comment (Quetzacoalt91) [inline controllers/admin/AdminThemesController.php]: It seems you could directly get the exporter:

```php
global $kernel;
$exporter = $kernel->getContainer()->get("prestashop.core.addon.theme.exporter");
```

Comment (maximebiloe) [inline controllers/admin/AdminThemesController.php]: Oh yeah! Nice catch 😉

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by aleeks** [comment]
Add a commit for windows hihi !
sans titre

**Comment by aleeks** [comment]
Need this by the way : https://github.com/PrestaShop/TranslationToolsBundle/pull/23

**Pull request merged**